### PR TITLE
Adding more rules to ananicy

### PR DESCRIPTION
**Pull request by Nightbane112**
Changes added :
- Added rule for SmoothVideo Project, 60 FPS motion interpolation software
- Added rule for Kwin Wayland session
- Added rule for KDE Connect
- Added rule for Steam client itself and its integrated web browser
- Added rule for Krita
- Added rule for PPSSPP, PSP Emulator (SDL version)
- Added rule for Wineserver, daemon for managing Windows apps via WINE

### CODE REVIEW

File: ananicy.d/00-default/wineserver.rules [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Reschedule Wineserver for better performance
+https://wiki.archlinux.org/index.php/Gaming#Optimus.2C_and_other_helping_programs
```
Comment (Nefelim4ag) [inline ananicy.d/00-default/wineserver.rules]: Missing comment #

Comment (Nightbane112) [reply ananicy.d/00-default/wineserver.rules]: Whoops! Must have deleted it by mistake. Sorry.

**Comment by Nefelim4ag** [comment]
@Nightbane112,
that's will be cool in general, if you rebase you patchset, to drop merge commits
Also that will be nice, if you try make one rule/file changes at one commit.

Any way, thanks, as i see you add some changes, i will wait 1-2 hours, and after can merge that by squash.

**Comment by Nightbane112** [comment]
@Nefelim4ag I'll try to fix those right away. I'm kinda new to this.

**Comment by Nefelim4ag** [comment]
@Nightbane112, np :)
git format-patch
git log
git reset
git am/apply

I think, that can help you

**Comment by Nightbane112** [comment]
Shall I close this commit as I had made changes to new commit in #45 as per your request?

**Comment by Nightbane112** [comment]
@Nefelim4ag Thanks! I just made a new commit as you requested here #45 . Hope you like it.

**Pull request closed without merging**